test/lib/search.bats

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bats
+
+load ../../lib/composure
+load ../../plugins/available/base.plugin
+
+cite _about _param _example _group _author _version
+
+load ../../lib/helpers
+load ../../lib/search
+
+NO_COLOR=true
+
+@test "helpers search aliases" {
+  run _bash-it-search-component 'plugins' 'base'
+  [[ "${lines[0]}" =~ 'plugins' && "${lines[0]}" =~ 'base' ]]
+}
+
+@test "helpers search all ruby et al" {
+  run _bash-it-search 'ruby' 'gem' 'bundle' 'rake' 'rails' '--disable'
+  run _bash-it-search 'ruby' 'gem' 'bundle' 'rake' 'rails'
+  [[ "${lines[0]/✓/}" == '      aliases  =>   bundler rails' ]]
+  [[ "${lines[1]/✓/}" == '      plugins  =>   chruby chruby-auto ruby' ]]
+  [[ "${lines[2]/✓/}" == '  completions  =>   bundler gem rake' ]]
```

Comment:
Good point — I did not realize that only the last one is evaluated for the overall test success. Joinining them with `&&` seems to do the trick.

However, adding `-chruby` uncovered one more broken use-case that I am fixing. Thanks for great suggestions!